docs/pages/QuickViews/QV-QuickViews.md

```diff
@@ -5,9 +5,9 @@ parent: QuickViews User Guide
 nav_order: 1
 ---
 
-# Quick Views
+# QuickViews
 {: .no_toc }
-The Quick Views allow you to create multiple views as Callouts, Elevantions and Sections of internal rooms and spaces.
+The QuickViews allow you to create multiple views as Callouts, Elevations and Sections of internal rooms and spaces.
```

Comment:
"With QuickViews, you can easily create Internal Elevation/Section Views and Callouts from Rooms/Spaces."